internals/cli/cmd_run.go

```diff
@@ -275,3 +283,109 @@ func convertArgs(args [][]string) (map[string][]string, error) {
 
 	return mappedArgs, nil
 }
+
+func maybeCopyPebbleDir(srcDir, destDir string) error {
+	if srcDir == "" {
+		return nil
+	}
+	dirEnts, err := os.ReadDir(destDir)
+	if err != nil && !errors.Is(err, os.ErrNotExist) {
+		return err
+	} else if len(dirEnts) != 0 {
+		// Skip non-empty dir.
+		return nil
+	}
+	fsys := os.DirFS(srcDir)
+	// TODO: replace with os.CopyFS
```

Comment:
This will almost certainly be included in Go 1.23, and I like to include that in the comment so we can search for "Go 1." later and find these to update:
```suggestion
	// TODO: replace with os.CopyFS when we're using Go 1.23
```